btk/draw_blends.py

```diff
@@ -468,31 +419,29 @@ def render_single(self, catalog_line, cutout, band):
         Returns:
             gal (galsim.InterpolatedImage): The galsim profile of a single galaxy
         """
-        k = int(np.random.rand(1) * len(self.cat))  # catalog_line["btk_index"][0]
-        gal = self.cat.makeGalaxy(k, gal_type="real", noise_pad_size=0).withFlux(1)
+        k = int(np.random.rand(1) * len(self.catalog))  # catalog_line["btk_index"][0]
+        cat = self.catalog.get
+        gal = self.catalog.makeGalaxy(k, gal_type="real", noise_pad_size=0).withFlux(1)
+        pix_stamp_size = int(self.stamp_size / survey.pixel_scale)
 
         # Convolution by a smal gaussian: The galsim models actally have noise in a little patch around them,
         # so gaussian kernel convolution smoothes it out.
-        # It haas the slight disadvantage of adding some band-limitedeness to the image,
+        # It has the slight disadvantage of adding some band-limitedeness to the image,
         # but with a small kernel, it's better than doing nothing.
-        gal = galsim.Convolve(gal, galsim.Gaussian(sigma=2 * cutout.pixel_scale))
+        gal = galsim.Convolution(gal, galsim.Gaussian(sigma=2 * survey.pixel_scale))
```

Comment:
Is there a reason for switching to `Convolution` instead of the more generic `Convolve` ?

Reply:
no reason thanks, will make sure we use Convolve from now on